Simplenote/Classes/SPTextView.m

```diff
@@ -48,6 +48,8 @@ - (instancetype)init {
         caret position.
      */
         self.textContainer.heightTracksTextView = NO;
+
+        self.keyboardDismissMode = UIScrollViewKeyboardDismissModeInteractive;
```

Comment:
Suggestion:

1. `SPNoteEditorViewController+Extensions.swift` new `configureTextView` method (there's a category that has a bunch of configure methods)
2. Set the dismissMode in the `EditorViewController` instead

Why: Since this is a concrete configuration, which depends on the way the component needs to be used, it'd be best to set in place.

Reply:
>Why: Since this is a concrete configuration, which depends on the way the component needs to be used, it'd be best to set in place.

Gotcha.  I moved it to SPNoteEditorViewController+Extensions.swift

One thing I am not sure about.  In the SPNoteEditorViewController.m file there is already an objective-c method for configureTextView.  I saw two options for setting this up and wasn't sure which to go with.
1. Create a new method on the swift file configureTextViewKeyboard which contains the change to the keyboard dismiss mode and then call that method when the view controller is setup.  This is the way I have done it currently because it is makes the fewest changes to how the app is currently setup
2. Or, I could see moving the entire configureTextViewKeyboard into the swift file instead.  If the longer term goal of SNiOS is to get as much of the app into swift as possible, this might be the better solution, but didn't want to make that change without discussing if this was the way we wanted to go.
3. Possible other option I haven't thought of

What do you think would be best?

Have added a commit that follows option one, which can be reviewed.